ui/webui/test/check-storage

```diff
@@ -234,6 +234,53 @@ class TestStorage(anacondalib.VirtInstallMachineCase):
         # (there is enough free space)
         s.check_partitioning_selected("use-free-space")
 
+    def testPartitioningObject(self):
+        # Test which partitioning object ends up being the AppliedPartitioning
+        # when we go back and forward the storage steps
+
+        b = self.browser
+        i = Installer(b, self.machine)
+        s = Storage(b, self.machine)
+
+        self.addCleanup(s.dbus_reset_partitioning)
+
+        # Read partitioning data before we do anything
+        created_partitioning = s.dbus_get_created_partitioning()
+
+        # Go to Review step
+        i.open()
+        i.next()
+        i.next()
+        i.next()
+        i.next()
```

Comment:
It seems to me that we might need to have a library function for reaching a given screen in the future, as the screen order ḿight be more dynamic in the future.